Ports/glib/package.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=glib
+version=2.69.0
+depends=libiconv
+useconfigure=true
+configopts="--cross-file ../cross_file.txt"
+files="https://gitlab.gnome.org/GNOME/glib/-/archive/2.69.0/glib-${version}.tar.gz glib-${version}.tar.gz 35af83aedf34b96f1e99ed9c995c50960509f504750fb4def2440a9406b00a95"
+auth_type=sha256
+
+configure()
+{
+    run meson _build $configopts
+}
```

Comment:
Please stick to the existing style

```suggestion
configure() {
    run meson _build $configopts
}
```